src/utils.py

```diff
@@ -159,41 +172,14 @@ def detect_block(device):
 
 
 def print_copyright(username):
-    if username is None or (
-        hashlib.sha1(username.encode("utf-8")).hexdigest() not in COPYRIGHT_BLACKLIST
-    ):
-        print_timeless(
-            "\nIf you like this script and want it to be improved, "
-            + COLOR_BOLD
-            + "donate please"
-            + COLOR_ENDC
-            + "."
-        )
-        print_timeless(COLOR_BOLD + "$3" + COLOR_ENDC + " - support this project")
-        print_timeless(COLOR_BOLD + "$10" + COLOR_ENDC + " - unblock extra features")
-        print_timeless(
-            COLOR_BOLD
-            + "$25"
-            + COLOR_ENDC
-            + " - same as $10 + vote for the next feature"
-        )
-        print_timeless("https://www.patreon.com/insomniac_bot\n")
-
-
-def print_blocked_feature(username, feature_name):
-    if hashlib.sha1(username.encode("utf-8")).hexdigest() not in COPYRIGHT_BLACKLIST:
-        print_timeless(
-            COLOR_FAIL
-            + "Sorry, "
-            + feature_name
-            + " is available for Patrons only!"
-            + COLOR_ENDC
-        )
-        print_timeless(
-            COLOR_FAIL
-            + "Please visit https://www.patreon.com/insomniac_bot\n"
-            + COLOR_ENDC
-        )
+    print_timeless(
```

Comment:
This will be removed with the other PR, in case it gets merged first - we need to remove it from yours too. We don't have donations right now and will probably keep it that way unless someone asks to donate or at least until we get some more features added. Right now, it would be silly to ask for donations.